Source/Generators/LMCPatchSignalGenerator.cc

```diff
@@ -61,15 +58,11 @@ namespace locust
 
     bool PatchSignalGenerator::Configure( const scarab::param_node& aParam )
     {
-        if( aParam.has( "filter-filename" ) )
-        {
-            gfilter_filename = aParam["filter-filename"]().as_string();
-        }
 
-        if( aParam.has( "filter-resolution" ) )
-        {
-            fFilter_resolution = aParam["filter-resolution"]().as_double();
-        }
+       if(!fReceiverFIRHandler.Configure(aParam,false))

```

Comment:
With the `FIRHandler` now changed to `FIRReceiverHandler`, there is no need to input false as a second argument to the Configure function in FIRHandler.